tests/e2e_test_auto_offer_on_device_rest.py

```diff
@@ -18,8 +18,8 @@
 args = parser.parse_args()
 connected_devices = []
 
-domain_name = 'https://d3aweb-dev.gridsingularity.com'
-websockets_domain_name = 'wss://d3aweb-dev.gridsingularity.com/external-ws'
+domain_name = 'https://d3aweb-staging.gridsingularity.com'
+websockets_domain_name = 'wss://d3aweb-staging.gridsingularity.com/external-ws'
```

Comment:
@Ravineesh Sorry, just saw that. I would have suggested to add these to variables as a argument to the script to easily be able to switch between domains from the d3a-e2e-ui-test side.

Reply:
Ok thanks. I'll change it when I'll switch it back to dev url.